docs/blockchain_data/confirmed_transactions.md

```diff
@@ -4,20 +4,29 @@ Up until this point, we've explored how to notify LDK of chain activity using
 blocks. But what if you're sourcing chain activity from a place that doesn't
 provide a block-centric interface, like Electrum?
 
-LDK has a `chain::Confirm` interface to support this use case, analogous to the
-block-oriented `chain::Listen` interface which we've been using up until now.
-With this alternative approach, you still need to give LDK block headers, but
-only for blocks containing transactions of interest. These are identified by
-`chain::Filter` as before. You also need to notify LDK of any transactions with
-insufficient confirmation that have been reorganized out of the chain. Use the
+LDK's `ChannelManager` and `ChainMonitor` have a `chain::Confirm` interface to
+support this use case, analogous to the block-oriented `chain::Listen`
+interface which we've been using up until now. With this alternative approach,
+you still need to give LDK information about chain activity, but only for
+transactions of interest. These are identified by
+`Confirm::get_relevant_txids`, as well as by `Filter`'s
+`register_tx`/`register_output` methods, just as before.
+
+You also need to notify LDK of any transactions with insufficient confirmations
+that have been reorganized out of the chain. Use `Confirm`'s
 `transactions_confirmed` and `transaction_unconfirmed` methods, respectively.
```

Comment:
Summing up my other comments, this seems less clear to me. We want to convey that:

(a) `transactions_confirmed` should be called for anything registered with `chain::Filter`
(b) `transaction_unconfirmed` should be called for anything returned by `Confirm::get_relevant_txids` that has been reorganized out of the chain

As written, the connection between those two pairs isn't obvious.

Reply:
Ah, thanks, good point. I reworded it to clarify this relationship of `Filter`/`transactions_confirmed` and `get_relevant_txids`/`transactions_unconfirmed`.